vre/templates/develops/develop_detail_5.html

```diff
@@ -0,0 +1,182 @@
+{% extends "base.html" %}
+{% load static from staticfiles %}
+{% block title %}Chicago 61 | VRE Mexico{% endblock %}
+{% block content %}
+{% include "sections_navbar.html" %}
+<main id="sections-content">
+  <div class="development-cover cover parallax" style="background-image: url('{% static "img/tmp/chicago_cover.jpg" %}')">
+    <img src="{% static "img/tmp/chicago_full_white_name.png" %}" alt="Chicago logo">
+  </div>
+  <div class="development-header-text">
+    Increíbles departamentos en el corazón<br>
+    de la colonia Nápoles, désde 62 m2.
+  </div>
+  <div class="gray-container development-download-brochure">
+    <div>
+      <span>Si quieres más información, descarga aquí nuestro brochure de ventas</span>
+      <form id="download-brochure-form">
+        {% csrf_token %}
+        <input type="email" id="download-brochure-email" placeholder="Correo electrónico" name="email">
+        <input type="hidden" name="source" value="chicago">
+	      <span id="download-brochure-span" style="display:block;"></span>
```

Comment:
Yes is a bad practice.

http://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css